.travis.yml

```diff
@@ -1,11 +1,12 @@
 language: node_js
-
 git:
   submodules: false
-
 os: linux
 dist: xenial
 sudo: enabled
-
 install:
-    - ./launch -d
+- "./launch -d"
+script:
+- npm run lint-check
+- npm test
```

Comment:
Shouldn't we enable test in separate PR?

Reply:
no, npm test is executed by default in .travis configuration 
when language: node_js is present in file
because I'm overwriting default behaviour for script section I need to include also default command
see details on: https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#default-build-script